en/Qtum-RPC-API/README.md

```diff
@@ -1,161 +1,211 @@
+# Qtum-RPC-API
+
+**This docs introce the rpc api method of Qtum. According to this docs you can learn how to use these api.** 
```

Comment:
typo: Introduce.

Need to indicate the Qtum core version.

Reply:
docs→doc
introduce→introduces
method→methods